scripts/rollup/modules.js

```diff
@@ -52,6 +52,17 @@ const devOnlyFilesToStubOut = [
   "'ReactTestUtils'",
 ];
 
+// Ordering of these imports is important;
+// The default import must follow deep imports or Rollup breaks.
```

Comment:
I gave an example below (in the comments) of what it maps the requires to but I'll update it to be a bit more explicit. Basically if it partial-matches one of the keys (eg `prop-types/checkPropTypes` matches `prop-types`) then it stops searching and uses a path like `prop-types/index.js/checkPropTypes.js`